corehq/motech/repeaters/templates/repeaters/partials/repeater_row.html

```diff
@@ -8,22 +8,22 @@
     </td>
     <td>
         <a href="{% url 'domain_report_dispatcher' domain report %}?repeater={{ repeater.repeater_id }}&amp;record_state=PENDING">
-            {{ repeater.get_pending_record_count }}
+            {{ repeater.count_State.Pending }}
```

Comment:
Slightly odd capitalization was chosen to make these match the corresponding IntegerChoices enum literals when searching with tools like grep.